### ISSUE TITLE
[css] invalid range or name for document symbol

### ISSUE THREAD

**Issue by reric88** [microsoft/vscode#176802]
Type: <b>Bug</b>

When working in a CSS document, if I use the `@media` query, a pop-up occurs stating:
```
"Request textDocument/documentSymbol failed.
Source: CSS Language Features (Extension)
```

I have not noticed any negative impact besides the pop-up obscuring terminal menu. When looking at the output, the log states:

```
Error - 10:37:39 AM] Request textDocument/documentSymbol failed.
Error: selectionRange must be contained in fullRange
	at t.validate (c:\Users\ruffy\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:94:25981)
	at new t (c:\Users\ruffy\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:94:26195)
	at V (c:\Users\ruffy\AppData\Local\Programs\Microsoft VS Code\resources\app\extensions\css-language-features\client\dist\node\cssClientMain.js:1:218165)
	at c (c:\Users\ruffy\AppData\Local\Programs\Microsoft VS Code\resources\app\extensions\css-language-features\client\dist\node\cssClientMain.js:1:256165)
	at t.map (c:\Users\ruffy\AppData\Local\Programs\Microsoft VS Code\resources\app\extensions\css-language-features\client\dist\node\cssClientMain.js:1:256251)
	at Object.asDocumentSymbols (c:\Users\ruffy\AppData\Local\Programs\Microsoft VS Code\resources\app\extensions\css-language-features\client\dist\node\cssClientMain.js:1:225983)
	at i (c:\Users\ruffy\AppData\Local\Programs\Microsoft VS Code\resources\app\extensions\css-language-features\client\dist\node\cssClientMain.js:1:164601)
	at async n.provideDocumentSymbols (c:\Users\ruffy\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:98:32515)
```

VS Code version: Code 1.76.1 (5e805b79fcb6ba4c2d23712967df89a089da575b, 2023-03-08T16:32:00.131Z)
OS version: Windows_NT x64 10.0.22621
Modes:
Sandboxed: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|AMD Ryzen 3 3250U with Radeon Graphics          (4 x 2595)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: disabled_off|
|Load (avg)|undefined|
|Memory (System)|5.92GB (0.81GB free)|
|Process Argv|--crash-reporter-id 11ca1b08-1b98-4518-a334-5fbb523ece23|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (11)</summary>

Extension|Author (truncated)|Version
---|---|---
better-comments|aar|3.0.2
prettier-vscode|esb|9.10.4
gml-support|lia|1.7.4
vscode-language-babel|mgm|0.0.37
git-graph|mhu|1.30.0
powershell|ms-|2023.2.1
vsliveshare|ms-|1.0.5834
vscode-css-variables-autocomplete|par|0.3.1
LiveServer|rit|5.7.9
code-spell-checker|str|2.19.0
vscode-icons|vsc|12.2.0

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
vslsvsres303:30308271
pythonvspyl392:30443607
vserr242cf:30382550
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vsdfh931:30280409
vshan820:30294714
vstes263:30335439
vscoreces:30445986
pythondataviewer:30285071
vscod805cf:30301675
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
cmake_vspar411:30581797
vsaa593:30376534
pythonvs932:30410667
cppdebug:30492333
vsclangdf:30486550
c4g48928:30535728
dsvsc012:30540252
pynewvextcfv2:30681851
azure-dev_surveyone:30548225
vsccc:30610678
pyindex848:30662994
nodejswelcome1cf:30587006
282f8724:30602487
pyind779:30671433
89544117:30613380
pythonsymbol12:30671437
a9j8j154:30646983
6233i204:30672705
vsccsb:30677849
vscodedisable:30660115
funwalk2cf:30682975
pythonms35:30680984

```

</details>

**Comment by Morpheous72** [comment]
Type: Bug

same probleme, working in a CSS document, using @media query, a pop-up occurs stating:

"Request textDocument/documentSymbol failed.
Source: CSS Language Features (Extension)

[Error - 19:22:19] Request textDocument/documentSymbol failed.
Error: selectionRange must be contained in fullRange
    at Function.validate (/Users/an0nym0us/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:94:25981)
    at new t (/Users/an0nym0us/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:94:26195)
    at V (/Users/an0nym0us/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/css-language-features/client/dist/node/cssClientMain.js:1:218165)
    at c (/Users/an0nym0us/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/css-language-features/client/dist/node/cssClientMain.js:1:256165)
    at Object.t.map (/Users/an0nym0us/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/css-language-features/client/dist/node/cssClientMain.js:1:256251)
    at Object.asDocumentSymbols (/Users/an0nym0us/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/css-language-features/client/dist/node/cssClientMain.js:1:225983)
    at i (/Users/an0nym0us/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/css-language-features/client/dist/node/cssClientMain.js:1:164601)
    at n.provideDocumentSymbols (/Users/an0nym0us/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:98:32515)

**Comment by marcandred-vortexsolution** [comment]
Same thing happeding to me. First seen using **@mixin** in a **.scss** file.

The popup seems to appears when it recognises a word. Right when the last letter is keyed in. After that, any non-letter caracters added or removed on the same line or any below will trigger the error.

Also, pasting **@mixin** makes the error popup appears, but **@mixin a** does not.

To be a little clearer, here's a list of patterns with **@mixin** that can be pasted in a **.scss** file to trigger the error : 

- `@mixin`
- `@mixin ` (added there's a space here at the end)
- `@mixin 1`
- `@mixin /`
- `@mixin/`
- `@mixin"`
- `@mixin(` (note: **@media(** does not trigger the error.)

**Comment by Ssong-Q** [comment]
Type: Bug

same problem here, working in a CSS document, using @media query, a pop-up and related terminal occurs stating:

`[Error - 오전 10:27:33] Request textDocument/documentSymbol failed.`
`Error: selectionRange must be contained in fullRange
    at Function.validate (/Users/james/Desktop/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:94:25981)
    at new t (/Users/james/Desktop/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:94:26195)
    at V (/Users/james/Desktop/Visual Studio Code.app/Contents/Resources/app/extensions/css-language-features/client/dist/node/cssClientMain.js:1:218165)
    at c (/Users/james/Desktop/Visual Studio Code.app/Contents/Resources/app/extensions/css-language-features/client/dist/node/cssClientMain.js:1:256165)
    at Object.t.map (/Users/james/Desktop/Visual Studio Code.app/Contents/Resources/app/extensions/css-language-features/client/dist/node/cssClientMain.js:1:256251)
    at Object.asDocumentSymbols (/Users/james/Desktop/Visual Studio Code.app/Contents/Resources/app/extensions/css-language-features/client/dist/node/cssClientMain.js:1:225983)
    at i (/Users/james/Desktop/Visual Studio Code.app/Contents/Resources/app/extensions/css-language-features/client/dist/node/cssClientMain.js:1:164601)
    at n.provideDocumentSymbols (/Users/james/Desktop/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:98:32515)`


<img width="1231" alt="스크린샷 2023-03-26 오전 10 32 42" src="https://user-images.githubusercontent.com/122101706/227750467-d0735c0a-db47-4da2-9111-f5a3f605a0ef.png">

**Comment by OneWithTheInternet** [comment]
I'm having the same issue.

**Comment by xoQueenSoph** [comment]
I'm confused. It says this was fixed but I'm still having the same issue. Nothing i've clicked on in this page is telling me how to fix it and use @media???

**Comment by xGeekoo** [comment]
Same issue

**Comment by pedrocostab** [comment]
Same issue

**Comment by aeschli** [comment]
As mentioned, the fix is in 1.78.
If you still see this, please add the build you are on and a small code sample to reproduce.
